### PR TITLE
[Security Content] Add Investigation Guides to Linux C2 Rules

### PR DESCRIPTION
**Pull request by Aegrah**
## Summary
This rule adds Investigation Guides for all of the current command and control related Linux rules. Next phase of IGs will focus on the next set.

### CODE REVIEW

File: rules_building_block/command_and_control_linux_proxychains_activity.toml [hunk 1]
```diff
@@ -21,11 +46,81 @@ index = ["logs-endpoint.events.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "ProxyChains Activity"
+note = """## Triage and analysis
+
+### Investigating ProxyChains Activity
+
+Attackers can leverage `proxychains` to obfuscate their origin and bypass network defenses by routing their malicious traffic through multiple intermediary servers.
+
+This rule looks for processes spawned through `proxychains` by analyzing proxychains process execution.
```
Comment (natasha-moore-elastic) [inline rules_building_block/command_and_control_linux_proxychains_activity.toml:55]:
```suggestion
This rule looks for processes spawned through `proxychains` by analyzing `proxychains` process execution.
```

File: rules/linux/command_and_control_cat_network_activity.toml [hunk 2]
```diff
@@ -19,9 +44,67 @@ index = ["logs-endpoint.events.*"]
...
+          - Check if the domain is newly registered or unexpected.
+          - Check the reputation of the domain or IP address.
+        - File access, modification, and creation activities.
 
-This rule requires data coming in from Elastic Defend.
+### Related Rules
+
+- Suspicious Network Activity to the Internet by Previously Unknown Executable - 53617418-17b4-4e9c-8a2c-8deb8086ca4b
+
+### False Positive Analysis
```
Comment (natasha-moore-elastic) [inline rules/linux/command_and_control_cat_network_activity.toml:85]:
```suggestion
### False positive analysis
```

File: rules/linux/command_and_control_cat_network_activity.toml [hunk 3]
```diff
@@ -19,9 +44,67 @@ index = ["logs-endpoint.events.*"]
...
+  - If scripts or executables were dropped, retrieve the files and determine if they are malicious:
+    - Use a private sandboxed malware analysis system to perform analysis.
+      - Observe and collect information about the following activities:
+        - Attempts to contact external domains and addresses.
+          - Check if the domain is newly registered or unexpected.
+          - Check the reputation of the domain or IP address.
+        - File access, modification, and creation activities.
 
-This rule requires data coming in from Elastic Defend.
+### Related Rules
```
Comment (natasha-moore-elastic) [inline rules/linux/command_and_control_cat_network_activity.toml:81]:
```suggestion
### Related rules
```

File: rules/linux/command_and_control_cat_network_activity.toml [hunk 4]
```diff
@@ -19,9 +44,67 @@ index = ["logs-endpoint.events.*"]
...
+This rule looks for a sequence of a `cat` execution event followed by a network connection attempt by the same `cat` process. 
+
+> **Note**:
+> This investigation guide uses the [Osquery Markdown Plugin](https://www.elastic.co/guide/en/security/master/invest-guide-run-osquery.html) introduced in Elastic Stack version 8.5.0. Older Elastic Stack versions will display unrendered Markdown in this guide.
+> This investigation guide uses [placeholder fields](https://www.elastic.co/guide/en/security/current/osquery-placeholder-fields.html) to dynamically pass alert data into Osquery queries. Placeholder fields were introduced in Elastic Stack version 8.7.0. If you're using Elastic Stack version 8.6.0 or earlier, you'll need to manually adjust this investigation guide's queries to ensure they properly run.
+
+#### Possible investigation steps
+
+- Identify any signs of suspicious network activity or anomalies that may indicate command and control activity or data exfiltration. This could include unexpected traffic patterns or unusual network behavior.
+  - Investigate listening ports and open sockets to look for potential protocol tunneling, reverse shells or data exfiltration.
```
Comment (natasha-moore-elastic) [inline rules/linux/command_and_control_cat_network_activity.toml:62]:
```suggestion
  - Investigate listening ports and open sockets to look for potential protocol tunneling, reverse shells, or data exfiltration.
```

**Review by natasha-moore-elastic — COMMENTED** [review]
LGTM, thanks! Left some minor editorial suggestions – some may apply to multiple rules.

**Review by terrancedejesus — APPROVED** [review]
Great work adding OSQuery to Linux investigation guides 🔥

**Comment by Aegrah** [comment]
Thanks @natasha-moore-elastic for the feedback. I applied most of it to the ruleset. Would you mind taking a quick second look, and approve if you agree with the changes?

**Review by natasha-moore-elastic — APPROVED** [review]

**Pull request merged**